frontend/lib/resolution/scope-queries.cpp

```diff
@@ -1179,7 +1234,7 @@ bool LookupHelper::doLookupInExternBlocks(const Scope* scope,
     }
   }
 
-  return true;
+  return LookupResult(found, /* nonFunctions; might be a lie; see above */ true);
```

Comment:
What am I supposed to see above?

Reply:
the comment about lying about `isParenfulFunction`; I'll update the code to make that clearer.